### ISSUE TITLE
Auto-close old issues where the reporter doesn't follow up

### ISSUE THREAD

**Issue by cvrebert** [twbs/bootstrap#14889]
Refs https://github.com/btford/mary-poppins/issues/21
Basically, have a bot that periodically checks for currently-open issues have had an "awaiting reply" tag for at least N days without any subsequent posts by non-twbs-team users, and closes them with a nice explanatory comment.

**Comment by kkirsche** [comment]
@cvrebert While testing in Bootstrap is good in regards to regressions, would this mean the preferred action would be to X-Ref the old issue from a new one stating that Component XYZ has regressed to with same issue XXXX instead of re-opening an existing issue for a regression?

Sorry if that didn't make sense. Basically just open a new issue for regressions or comment and reopen old ones if the issue arises again

**Comment by cvrebert** [comment]
@kkirsche Yes. A new issue is a *much* better way to get our attention as maintainers, and folks sometimes incorrectly think a new problem is related to an old issue because the symptoms might be kinda similar, when in fact the underlying cause ends up being unrelated, in which case reopening would be semantically wrong.

**Comment by mdo** [comment]
Does this need to be open here, or could we perhaps use another public repo to track these kind of meta things? Things like new bots, org level changes, etc.

**Comment by cvrebert** [comment]
I am happy to report that this is now nearly done. I am mostly waiting on some fixes to an upstream library that will allow my project to have thorough unit tests for its GitHub interactions.

Refs: https://github.com/jcabi/jcabi-github/issues/1055, https://github.com/jcabi/jcabi-github/issues/1056, https://github.com/jcabi/jcabi-github/pull/1063

**Comment by cvrebert** [comment]
Blocked on one last PR: https://github.com/jcabi/jcabi-github/pull/1068

**Comment by cvrebert** [comment]
(sigh) Had to bifurcate that PR, so there's still one to go: https://github.com/jcabi/jcabi-github/pull/1073

**Comment by cvrebert** [comment]
Yay, the bot is finally ready! I hope to deploy it soon, pending discussion and final approval from the other members of the Core Team.

The auto-locking feature has been punted to https://github.com/cvrebert/no-carrier/issues/3
